plugins/wacom-usb/fu-wac-module.c

```diff
@@ -182,6 +182,8 @@ fu_wac_module_set_feature(FuWacModule *self,
 				       fu_wac_module_command_to_string(command));
 			return FALSE;
 		}
+		if (priv->status != FU_WAC_MODULE_STATUS_OK)
+			return FALSE;
```

Comment:
need to set an error if we return FALSE.

Reply:
ACK